src/api/app/controllers/webui/staging/workflows_controller.rb

```diff
@@ -38,6 +40,7 @@ def show
     @ignored_requests = @staging_workflow.ignored_requests.first(5)
     @more_ignored_requests = @staging_workflow.ignored_requests.count - @ignored_requests.size
     @empty_projects = @staging_workflow.staging_projects.without_staged_requests
+    @managers = @staging_workflow.managers
```

Comment:
I don't like the naming managers. With this naming I would expect I get a list of Users but what I get is a single group. Doesn't make sense. So either renaming it to managers_group or really return the managers.

Reply:
Same is true for the assocation

Reply:
**Group**. *Noun*. Any collection or assemblage of persons or things; cluster; aggregation.

So manager**s** suggests this is a group :-) And please keep in mind that in the future this should be multiple groups and users.

Reply:
@hennevogel  I don't get it ... It is a ``has_one`` assocation so plural does not make sense here, using plural for a belongs_to / has_one assocation violates convention.

Reply:
This is a name. A name that should generate meaning if you read this code. This name is not there so you can guess what objects it returns. Who says that `staging_workflow.managers` does not return a comma separated list of names or any other data structure that you can cook up?

Reply:
What? No, there are conventions you should follow for naming. And yes it should be possible from the name to know if you get one object or several (and even to know what type it is e.g. ``project`` -> object vs ``project_name`` -> string). 

> belongs_to associations must use the singular term. If you used the pluralized form in the above example for the author association in the Book model and tried to create the instance by Book.create(authors: @author), you would be told that there was an "uninitialized constant Book::Authors". This is because Rails automatically infers the class name from the association name. If the association name is wrongly pluralized, then the inferred class will be wrongly pluralized too.

https://edgeguides.rubyonrails.org/association_basics.html#the-belongs-to-association

Reply:
> be limited to one group in the future.

If this is the case, we have to change the whole assocation anyway in the future and everything what is using it.

> Which wont happen because we use

Yes, for this special case not. But does it mean just because we don't use this feature of Rails we ignore conventions? Conventions make it easier to work with the code. Do we now name controllers singular because we can hardcode the name in the routes? Of course not! This will cause somebody sometime to debug in the future because the association is not behaving as expected... Not mentioning that you don't get what you expect. Why are we ignoring conventions without having a reason?

But seems like I'm fight one's last stand, so ending this conversation then.

Reply:
I think I will change it to `managers_group` to follow Rails convention and we can change it afterwards if turns into a polymorphic association with User and/or Group